<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-2RK5HRGB" failures="1" tests="2" name="Test" time="28.287" errors="0" timestamp="2021-08-13T18:56:53 IST">
  <testcase classname="com.mainProject.testCases.TC_LoginTest_001" name="loginTest" time="3.945"/>
  <testcase classname="com.mainProject.testCases.fbLogintTest" name="fbLogintest" time="1.546">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.mainProject.testCases.fbLogintTest.fbLogintest(fbLogintTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- fbLogintest -->
</testsuite> <!-- Test -->
